README.md

```diff
@@ -12,6 +12,9 @@ The integration is usable with Kentico Xperience 13 **hotfix 13.0.142** or newer
 ## Installation
 
 1. Install the [Kentico.Xperience.OpenAI.Azure.KX13](https://www.nuget.org/packages/Kentico.Xperience.OpenAI.Azure.KX13) NuGet package in the administration project.
+    - If you encounter an *Unable to resolve dependency 'Azure.AI.OpenAI'...* error during installation, consider a console installation through **Package Manager Console** (*Tools* → *NuGet Package Manager* → *Package Manager Console* in Visual Studio) and install the package with the **-IncludePrerelease** flag:
+        
+        `Install-Package Kentico.Xperience.OpenAI.Azure.KX13 -IncludePrerelease`
```

Comment:
This doesn't necessarily needs to be on a separate line, especially since it's a simple code macro (and not a proper code block. Therefore either:
- move this to the end of the preceding paragraphs
- or make this a proper markdown code block using '```' (check https://www.codecademy.com/resources/docs/markdown/code-blocks)